client/signup/tailored-flow-processing-screen/index.jsx

```diff
@@ -55,6 +55,14 @@ export default function TailoredFlowProcessingScreen( { flowName } ) {
 	const progress = ( currentStep + 1 ) / totalSteps;
 	const isComplete = progress >= 1;
 
+	// Temporarily override document styles to prevent scrollbars from showing
```

Comment:
`html` is styled by default with visible overflow. Not overriding it will result in a visible and disabled vertical scrollbar on some browsers. I think that first impressions matter and I added this to remove it temporarily.